mixin/dashboards/ibm-db2-overview.libsonnet

```diff
@@ -0,0 +1,874 @@
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'percent',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '10.0.1-cloud.2.a7a20fbf',
+};
+
+local logOperationsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'increase(ibm_db2_log_operations_total{job=~"$job",instance=~"$instance",database_name=~"$database_name"}[$__interval:])',
+      datasource=promDatasource,
+      legendFormat='{{database_name}} - {{log_member}} - {{log_operation_type}}',
+      interval='1m',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Log operations',
+  description: 'The number of log pages read and written to by the logger.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+{
+  grafanaDashboards+:: {
+    'ibm-db2-overview.json':
+      dashboard.new(
+        'IBM DB2 overview',
+        time_from='%s' % $._config.dashboardPeriod,
+        tags=($._config.dashboardTags),
+        timezone='%s' % $._config.dashboardTimezone,
+        refresh='%s' % $._config.dashboardRefresh,
+        description='',
+        uid=dashboardUid,
+      )
+
+      .addTemplates(
+        std.flattenArrays([
+          [
+            template.datasource(
+              promDatasourceName,
+              'prometheus',
+              null,
```

Comment:
Normally, our mixins/integration have datasource variables available (visible) to users. I think it is better to show loki and prometheus datasources as variables on top.

Also, by convention, if you have two datasources(metrics + logs) on the same dashboards in variables, they should be named:
- prometheus_datasource
- loki_datasource

Reply:
Hey @v-zhuravlev, so I looked into your comment and came up with a few things. 

- I believe the datasource selectors are present currently. When I run "make dashboards_out" and import the resulting json file into the shared cloud instance the datasource selectors appear at the top. If you wanna check it out, it should be under the "Integration - IBM DB2" dashboard folder in the shared cloud instance. 
- I'm also pretty sure that the variables are named "prometheus_datasource" and "loki_datasource". On line 798 theres a ```promDatasourceName``` referenced, and on line 9 it assigns that variable with "prometheus_datasource". So I think that's how it's naming the prometheus data source, which is how I saw it named using the json file generated by dashboards_out. A similar thing occurs with logs (lines 807 and 10).
- Not sure if this is what caught your eye, but the screenshots I've included are lacking the datasource selectors. I can get some better screenshots and send them your guy's way and have them replace the pictures that've been uploaded already.

Let me know your thoughts and if I missed anything! I think I addressed your concern with the datasource selectors, but please let me know if I missed anything. Thanks!

Reply:
You are right, I think I got confused by the screenshot. Sorry about that!